core/controllers/dashboard_test.py

```diff
@@ -463,6 +463,103 @@ def test_no_explorations(self):
         self.assertEqual(response['explorations_list'], [])
         self.logout()
 
+    def test_view_dashboard_no_explorations(self):
+        self.login(self.OWNER_EMAIL)
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+
+        # Testing that creator only visit dashboard without any exploration created
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),0)
+        
+
```

Comment:
Don't use so many newlines in between. Please take a look at how the existing code in this file (and the whole repository for that matter) is structured.